### ISSUE TITLE
Should fileCache store file in the cache folder instead of the document folder?

### ISSUE THREAD

**Issue by wcandillon** [wkh237/react-native-fetch-blob#253]
I was surprised to see that the option fileCache: true store the images in the document folder. Wouldn't it be more intuitive to put it into the cache folder?

**Comment by wkh237** [comment]
@wcandillon , yeah I think you're right, but the change will break backward compatibility, we can add this one to `v0.11.0` milestone.
